prepqueue.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+det e create $PROJECT_ROOT/e2e_tests/tests/fixtures/no_op/adaptive.yaml $PROJECT_ROOT/e2e_tests/tests/fixtures/no_op > /dev/null &
```

Comment:
This is interesting...why do we have to do this to prep for queue?

Reply:
ahh it's just a convenience for populating the queue when developing. will go away before landing in master.